src/Blazored.Toast/Configuration/ToastSettings.cs

```diff
@@ -5,22 +5,56 @@ namespace Blazored.Toast.Configuration;
 
 public class ToastSettings
 {
+    /// <summary>
+    /// Add additional classes that will be applied to toast component
+    /// </summary>
     public string AdditionalClasses { get; set; }
+    /// <summary>
+    /// Icon name, currently supporeted names for FontAwesome icons, Material icons
+    /// </summary>
     public string? Icon { get; set; }
+    /// <summary>
+    /// 
+    /// </summary>
     public IconType? IconType { get; set; }
+    /// <summary>
+    /// When enabled progress will be shown until toaster is closed based on  <see cref="Timeout"/> value
+    /// </summary>    
     public bool ShowProgressBar { get; set; }
+    /// <summary>
+    /// When enabled and mouse is over the toast, timeout period will be paused.
+    /// Use this with <see cref="ExtendedTimeout"/>
+    /// </summary>
+    public bool PauseProgressOnHover { get; set; }
+    /// <summary>
+    /// 
+    /// </summary>
     public bool ShowCloseButton { get; set; }
+    /// <summary>
+    /// 
+    /// </summary>
     public Action? OnClick { get; set; }
+    /// <summary>
+    /// Timeout in seconds for toaster to close
+    /// </summary>
     public int Timeout { get; set; }
-    
+    /// <summary>    
+    /// When <see cref="PauseProgressOnHover"/> is enabled, set ExtentedTimeout in seconds
+    /// to close toast after mouse loose focus.
+    /// Default value is 0, meaning no extended timeout
+    /// </summary>
+    public int ExtendedTimeout { get; set; }
```

Comment:
Thank you for adding the summaries for these methods. If we're going to have them though, can we have a summary for all settings. 

Nit pick: Can you also make sure there a line between each property the summary of the next property. Without any line breaks, things are much harder to read.

Reply:
i have replaced summaries, english is not ma natural language but i did my best, maybe is too much so let me know if you think this is ok?

Btw, this is my first PR ever so feel free to correct me and teach me if i am doing something wrong, if you have time of course :)

![BlazoredToast_summaryRemarks](https://user-images.githubusercontent.com/506853/219739117-6ab3e835-913c-4b44-a548-d1d56d13bac9.png)

Reply:
These are great! Thank you.